### ISSUE TITLE
Expose Action helper types

### ISSUE THREAD

**Issue by ChandlerBaskins** [ngrx/platform#4059]
### Which @ngrx/* package(s) are relevant/related to the feature request?

store

### Information

Currently there are very little typings exposed in the public api that support the Action concept. Is it possible to expose the stable types in the public API? My use case is to add references to Actions on Objects that I normalize. I deal with several different apis and several feature stores and aggregate all this data into one array. Having the ref to the action group is useful so I don't have to normalize to a group type and maintain a map of these actions (which you can't type anyways).  Also would love if you can somehow pass in a string definition to force you action group to have an action of that name. 

TLDR

- Expose ActionGroup in public api
- Accept generic arg in `createActionGroup` where dictionary of events contains the string. 

Edited: typos

### I would be willing to submit a PR to fix this issue

- [X] Yes
- [ ] No

**Comment by markostanimirovic** [comment]
Hi @ChandlerBaskins,

Can you provide a code example of what you would like to achieve?

**Comment by markostanimirovic** [comment]
I'm going to close this issue since the requested example was not provided for:

> Accept generic arg in createActionGroup where dictionary of events contains the string.

The feature request for:

> Expose ActionGroup in public API

is already opened: #3811
